projects/plugins/jetpack/_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -863,21 +866,36 @@ public function update_data( $request ) {
 					break;
 
 				case 'admin_bar':
-				case 'enable_calypso_stats':
 				case 'roles':
 				case 'count_roles':
 				case 'blog_id':
-				case 'do_not_track':
```

Comment:
What's the benefit of this change? It appears out of scope and has a potential for regression.

Reply:
True. I'll revert the change here, and only move `enable_calypso_stats` to limit the scope.

Reply:
Changes reverted in fd309ea265